modules/mill-codegen-plugin/test/src/smithy4s/codegen/mill/SmithyLSPConfigSpec.scala

```diff
@@ -0,0 +1,58 @@
+package smithy4s.codegen.mill
+
+import mill._
+import mill.scalalib._
+import mill.testkit.MillTestKit
+import sourcecode.FullName
+
+class SmithyLSPConfigSpec extends munit.FunSuite {
+
+  private object testKit extends MillTestKit
+
+  test("config gets generated") {
```

Comment:
I did not write the test because I was not sure how to, that `root` trick does it perfectly

thanks for stepping up